frontend/package.json

```diff
@@ -12,6 +12,7 @@
   "private": true,
   "dependencies": {
     "@angular/animations": "~12.1.0",
+    "@angular/cdk": "^12.1.4",
```

Comment:
Why do you need this?

Reply:
it was needed for primeng/dropdown. ok, I remove it.

Reply:
![error](https://user-images.githubusercontent.com/66973922/127776074-5046b35f-2365-4d94-8bf2-ada19969e04e.png)